src/Core/Content/Category/SalesChannel/NavigationRoute.php

```diff
@@ -55,28 +55,40 @@ public function getDecorated(): AbstractNavigationRoute
      * @OA\Post(
      *      path="/navigation/{requestActiveId}/{requestRootId}",
      *      summary="Fetch a navigation menu",
-     *      description="This endpoint returns a categories that can be used as a page navigation. You can either return them as a tree or as a flat list. You can also control the depth of the tree.",
+     *      description="This endpoint returns categories that can be used as a page navigation. You can either return them as a tree or as a flat list. You can also control the depth of the tree.
+    Instead of passing uuids, you can also use one of the following aliases for the activeId and rootId parameters to get the respective navigations of your sales channel.
+
+     * main-navigation
+     * service-navigation
+     * footer-navigation",
      *      operationId="readNavigation",
      *      tags={"Store API", "Category"},
      *      @OA\Parameter(name="Api-Basic-Parameters"),
      *      @OA\Parameter(
      *          name="requestActiveId",
-     *          description="The active category id",
+     *          description="Id of the active category in the navigation tree (if not used, just set to the same as rootId).",
      *          @OA\Schema(type="string", format="uuid"),
```

Comment:
Even though this is technically correct, API generators will punish us for that documentation, because the UUID format referred to in here contains `-` characters. In order to reflect the actual values that our API works with, we should replace every occurrence of UUIDs with

```php
// @OA\Schema(type="string", pattern="^[0-9a-f]{32}$")
```